releasenotes/notes/42055.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: release-notes/v2
+
+kind: bug-fix
+
+area: installation
+
+issue:
+- 42055
+
+releaseNotes:
+- **Modified** the default directory that OpenShift is installed to from kube-system to istio-operators. Please see the OCP platform documentation for further information.
```

Comment:
Typo here, the namespace was changed to `openshift-operators` not `isto-operators`.